packages/fxa-settings/src/components/UnitRow/index.tsx

```diff
@@ -51,7 +51,13 @@ export const UnitRow = ({
       <div className="unit-row-header">
         <h3 data-testid="unit-row-header">{header}</h3>
       </div>
-      <div className="unit-row-content">
+      {/* apply 'truncate' to the parent if 'headerValueClassName' contains it */}
```

Comment:
truncation tends to lead to interesting edge cases in German locales. just curious, do we have storybook plans for specific localizations?

Reply:
We don't, but that's a _really_ great suggestion - we could have one for German and another for rtl languages. I've added a note for myself, I'll get a ticket filed in settings v2 for these two cases (let me know if there should be more), 'cause actually, we truncate it preeettty short right now:

Content server: 
<img width="713" alt="image" src="https://user-images.githubusercontent.com/13018240/91235614-84f43c80-e6fb-11ea-82ff-daeeefb37033.png">

With this PR:
<img width="1226" alt="image" src="https://user-images.githubusercontent.com/13018240/91235655-9a696680-e6fb-11ea-9ffb-3975c5ad6596.png">